### PR TITLE
[#145965563] Add Occupancy Dashboard Tab

### PR DESCRIPTION
**Pull request by waggles**
* Adds new tab, updates link collection to point to this url instead
* ditches product page tab (used to be next to Card Readers)
* partial isn't fully complete yet, will match existing version once I find the photo

### CODE REVIEW

File: vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb [hunk 1]
```diff
@@ -33,11 +33,11 @@ def self.orphaned
     end
 
     def self.current
-      valid.where(exit_at: nil).where.not(entry_at: nil)
+      valid.order(entry_at: :desc).where(exit_at: nil).where.not(entry_at: nil)
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb]: I think the `current` scope does the `where`s.

Comment (jhanggi) [inline vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb]: Silly me. This _is_ the scope.

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb [hunk 2]
```diff
@@ -25,6 +25,10 @@ def initialize
       @active_tab = "admin_occupancies"
     end
 
+    def dashboard
+      @secure_rooms = current_facility.products(SecureRoom).active
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb]: We default all of our secure rooms to `is_hidden`. I think we need `active_plus_hidden`.

Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb]: And we should sort them alphabetically.

Comment (waggles) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb]: what does hidden mean exactly, business case-wise?

Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb]: Hidden means it's hidden from normal users. So like this view: http://localhost:3000/facilities/example

File: vendor/engines/secure_rooms/app/views/secure_rooms/shared/_secure_rooms_dashboard.html.haml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+-# TODO: Styling needs work
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/views/secure_rooms/shared/_secure_rooms_dashboard.html.haml]: I think this looks good enough for a V1. We can adjust the styling as we do the 📺  version.

Comment (waggles) [inline vendor/engines/secure_rooms/app/views/secure_rooms/shared/_secure_rooms_dashboard.html.haml]: I'll add that story number to this comment

**Comment by jhanggi** [comment]
Not directly associated with this story, but the "In progress" occupancies tab should probably get a count. I'd be fine if it's calculated as part of the page load rather than the tab_count ajax.

**Comment by waggles** [comment]
Added story [#146119365] https://www.pivotaltracker.com/story/show/146119365

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb [hunk 4]
```diff
@@ -25,6 +25,10 @@ def initialize
       @active_tab = "admin_occupancies"
     end
 
+    def dashboard
+      @secure_rooms = current_facility.products(SecureRoom).active_plus_hidden.order(name: :asc)
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb]: We have a `alphabetized` scope.

**Pull request merged**